backend/de.metas.util/src/main/java/de/metas/util/lang/ClassLoaderUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * de.metas.util
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.util.lang;
+
+import de.metas.util.Check;
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
+import javax.annotation.Nullable;
+
+@UtilityClass
+public class ClassLoaderUtil
+{
+	public Class<?> validateJavaClassname(
```

Comment:
and now, does it make sense to have this method here.... or we can move it in Util, near the newInstance?

... and when we will decide to have a separate utils class for those, we can move them all together.
wdyt?

Reply:
moved.